calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx

```diff
@@ -16,47 +18,38 @@ Learn about the new features, bug fixes, and other updates in this release of {{
 
 ## New features and enhancements
 
-### Improved flow log filtering for destination domains
-
-We’ve updated the Felix parameter (`dest_domains`) for DNS policy to make it easy to find only domain names that the deployment actually connected to (not all the domain names that got translated to the same IP address).
-For more information, see [Flow log data types](../visibility/elastic/flow/datatypes.mdx).
-
-### New flow logs panel on Endpoints page
-
-We've updated the Endpoints page in Manager UI with a new flow logs panel so you can view and filter Endpoints associated with denied traffic. Flow log metadata includes the source, destination, ports, protocols, and other key forms. We've also updated the Policy Board to highlight policies with denied traffic.
-
-### Improvements to security events dashboard
+### Support for OpenShift hosted control planes
 
-We've added the following improvements to the [Security events dashboard](../threat/security-event-management.mdx):
+You can now install Calico Enterprise on OpenShift clusters that use hosted control planes.
 
-- Jira and Slack webhook integration for security event alerts
+For more information, see [Installing Calico Enterprise on an OpenShift HCP cluster](../getting-started/install-on-clusters/openshift/hostedcontrolplanes.mdx)
 
-   By [configuring webhooks](../threat/configuring-webhooks.mdx), you can now push alerts from the Security Overview dashboard in Manager UI to Jira and Slack so incident response and security teams can use native tools to respond to security event alerts.
+### Packet capture and compliance reports are disabled by default
 
-- Added threat feed alerts
+We've changed the default behavior for packet capturing and compliance reporting.
+These features are now disabled by default.
+You can enable either feature in Manager UI or by running a kubectl command.
```

Comment:
Nit: It is good to say, they can enable during installation too -
`You can enable either feature during installation, later via the Manager UI, or by running a kubectl command.` but it is ok even if we dont state them explicilty. I will leave that up to you.